data/2022-results.json

```diff
@@ -4,5 +4,10 @@
   "total-number-responses": 123,
   "total-trip-count": 100,
   "total-distance": 12300,
-  "total-co2-emissions-tonnes": 8888
+  "total-co2-emissions-tonnes": 8888,
+  "work-mode": {
+    "onsite": 5,
+    "hybrid": 50,
+    "wfh": 20
```

Comment:
if we provide more context and order the work arrangement data here, there won't need to do any further processing in `WorkArrangement.jsx` using `Object.entries(surveyData).reduce(...)` to obtain the results we need to display on the results page.
For example, here are 3 examples on how we could provide more context and order the data (from largest-to-smallest values):

Option 1: use an array of objects, with items sorted by value:
```suggestion
  "work-arrangement-proportions": [
    { type: "hybrid", value: 50 },
    { type: "wfh", value: 20 },
    { type: "onsite", value: 5 }
  ]
```

Option 2: use an array of objects (sorted), with percentage as values:
```suggestion
  "work-arrangement-proportions": [
    { type: "hybrid", percent: 67 },
    { type: "wfh", percent: 27 },
    { type: "onsite", percent: 6 }
  ]
```

Option 3: use an a nested object (of objects), and be more explicit of the order:
```suggestion
  "work-arrangement-proportions": {
   1st: { type: "hybrid", percent: 67 },
   2nd: { type: "wfh", percent: 27 },
   3rd: { type: "onsite", percent: 6 }
  }
```

Option 4: use a plain old object of sorted items:
```suggestion
  "work-arrangement-proportions": {
    "hybrid": 50,
    "wfh": 20,
    "onsite": 5
  }
```

the data model in Option 4 is probably closest to what you have implemented in PR, except that it is ordered. That means you don't need to chain `Object.entries(surveyData)` with `reduce()`.